core/baseline-core-v1.0-psd01.md

```diff
@@ -2786,113 +2795,722 @@ The document defines storage capability requirements in the following areas:
+4. Verify that the BPI grants access to the BPI Subject after successful authentication with username and password.
+5. Attempt to access a protected resource or perform a protected action within the BPI.
+6. Verify that the BPI grants access to the BPI Subject if the BPI Subject has appropriate permissions.
+7. Attempt to enable MFA for the BPI Subject account used to log in to the BPI.
+8. Verify that the BPI supports multiple factors for MFA, such as SMS, email, or an authenticator app.
+9. Attempt to log out and log back in to the BPI using only the username and password credentials.
+10. Verify that the BPI requires the BPI Subject to provide an additional factor of authentication for MFA after the initial login.
+11. Attempt to access a protected resource or perform a protected action within the BPI.
+12. Verify that the BPI grants access to the BPI Subject if the BPI Subject has appropriate permissions and has provided the required additional factor of authentication for MFA.
+13 Attempt to disable MFA for the BPI Subject account used to log in to the BPI.
+14 Verify that the BPI disables MFA for the BPI Subject account and allows the BPI Subject to log in with only username and password credentials.
+
+Test Passing Criteria: The test will pass if
+
+* The BPI redirects the BPI Subject to a login page or displays a message indicating that authentication is required.
+*Successful login using username and password credentials grants access to the BPI.
+*Access to protected resources or actions within the BPI is granted only to users with appropriate permissions.
+*The BPI supports multiple factors for MFA, such as SMS, email, or an authenticator app.
+*After enabling MFA, the BPI requires the BPI Subject to provide an additional factor of authentication for MFA after the initial login.
+*Access to protected resources or actions within the BPI is granted only if the BPI Subject has provided the required additional factor of authentication for MFA.
+*After disabling MFA, the BPI allows the BPI Subject to log in with only username and password credentials.
+
 #### **[R287]**	
 BPI Storage MUST support hardware security modules (HSM). 
 
 *This document adopts the [NIST definition](https://csrc.nist.gov/glossary/term/hardware_security_module_hsm) and for further information, refer to [[HSM](#hsm)].*
 
+[[R287]](#r287) Testability:
+
+Preconditions:
+
+* A BPI test environment is installed and configured properly.
+* The BPI storage component is designed to support an HSM.
+* The HSM is installed and properly configured to work with the BPI storage component.
+* A BPI Subject has appropriate permissions to access the BPI.
+
+Test Steps:
+
+1. Attempt to access the BPI's user interface or API without providing any authentication credentials.
+2. Verify that the BPI redirects the BPI Subject to a login page or displays a message indicating that authentication is required.
+3. Enter valid username and password credentials to log in to the BPI.
```

Comment:
```suggestion
3. Log in to the BPI.
```
(There are requirements on how you do this, but the details shouldn't influence the outcome of this test)